cast/java/test/data/src/testSubjects/java/javaeight/ParamsAndCapture.java

```diff
@@ -0,0 +1,57 @@
+package javaeight;
+
+public class ParamsAndCapture {
```

Comment:
@juliandolby I previously wrote this test for lambda support in the CHA call graph, and it seems to be failing here.  It crashes with an NPE in the `com.ibm.wala.cast.java.translator.jdt.JDT2CAstUtils#anonTypeName` method.  Could you take a look?